src/main/java/duke/model/meal/Breakfast.java

```diff
@@ -15,5 +15,6 @@
     public Breakfast(String description, LocalDate date, HashMap<String, String> details, String costStr) {
         super(description, date, details, costStr);
         super.type = "B";
```

Comment:
Can consider removing the old type since the new mealType is parameterized (better). But just need to double check that rest of code works properly after removing it. Probably only the Meal toString uses the type.

Reply:
Noted!